doc/release/1.13.0-notes.rst

```diff
@@ -83,6 +83,9 @@ for instance). Note that this does not remove the need for Mingwpy; if you make
 extensive use of the runtime, you will most likely run into issues_. Instead,
 it should be regarded as a band-aid until Mingwpy is fully functional.
 
+The MinGW toolset can now also be used together with the moveable WinPython 3.4
+distribution, for programs that uses a PySide Qt4 front-end.
```

Comment:
This sentence has nothing ``numpy`` / ``numpy.distutils`` specific in it, so maybe make a bit more specific as to what use case is now enabled?